devise_token_auth.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.2.0"
 
   s.add_dependency 'rails', '>= 4.2.0', '< 6.2'
-  s.add_dependency 'devise', '> 3.5.2', '< 5'
+  s.add_dependency 'devise', '~> 4.7', '< 5'
```

Comment:
Deprecating all the previous devise versions?

Reply:
code reverted